crowdsec-docs/docs/appsec/intro.md

```diff
@@ -0,0 +1,46 @@
+---
+id: intro
+title: Introduction
+sidebar_position: 1
+---
+
+# Introduction
+
+<!-- xx : fix crowdsec version -->
+The application security component available in crowdsec:
+
+ - Provides low-effort virtual patching capabilities
+ - Supports your legacy ModSecurity rules
+ - Combines the advantages of classical WAF with the enhanced crowdsec capabilities to allow the detection of behaviors that would otherwise be difficult
+ - Is fully integrated with the Crowdsec software stack, such as the console and remediation components.
+
+<!-- xx :  links -->
+The application security component takes advantage of the remediation components existing in web servers (such as Nginx, Traefik, Haproxy etc.) to provide web application firewall capabilities:
```

Comment:
[proposition] This component capitalizes on existing remediation functions in web servers (such as Nginx, Traefik, Haproxy, etc.) to provide web application firewall capabilities.